### ISSUE TITLE
[$2000] IOU - Add bank account is looping when clicking pay with expensify

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#20280]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Have an account (under the wallet beta) with a silver/pending wallet. Note: every account under the wallet beta has a silver wallet created automatically.
2. If this account does not have a personal bank account (PBA) set up, follow the below steps:
3. Log into NewDot and navigate to Settings > Payments > Add payment method > Bank account
4. Select Connect via plaid and use the following credentials:
Wells Fargo
Username: user_good
password: pass_good
Mobile verification code: credential_good
Plaid saving
5. Have another user request money to the user you are testing with
6. Attempt to pay the money request
## Expected Result:
User able to proceed with payment as bank account already added

## Actual Result:
Add bank account is looping when pay with expensify clicked

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** v1.3.24-4

**Reproducible in staging?:** Yes
**Reproducible in production?:** Cannot test in prod

**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/44479856/06ec81b6-0540-45fd-9b89-fe15a1ebdc71

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal team
**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

Issue found while executing QA for https://github.com/Expensify/App/pull/19753
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014f5fed1085f06532</li>
        <li>Upwork Job ID: 1668322040539529216</li>
        <li>Last Price Increase: 2023-06-20</li>
    </ul>
</details>

**Comment by isagoico** [comment]
Issue is failing PR https://github.com/Expensify/App/pull/19753 from the checklist deploy

**Comment by Christinadobrzyn** [comment]
Following this - https://github.com/Expensify/App/issues/17027#issuecomment-1584842843

**Comment by petromoldovan** [comment]
@isagoico please check if credentials for the Wells Fargo are correct.
![Screenshot 2023-06-13 at 17 39 28](https://github.com/Expensify/App/assets/15109844/b1ccdf3a-7843-4fae-87db-dc4b7f3bacd0)

**Comment by s77rt** [comment]
Not overdue, still looking for proposals and I think this may need reproduction confirmation

**Comment by Christinadobrzyn** [comment]
I think we might be waiting on @Skalakid to look into this based on - https://github.com/Expensify/App/issues/17027#issuecomment-1584842843

**Comment by stitesExpensify** [comment]
We are no longer waiting FYI. It is not related to that PR

**Comment by Christinadobrzyn** [comment]
Looks like we're waiting on proposals!

**Comment by s77rt** [comment]
Not overdue. Looking for proposals

**Comment by s77rt** [comment]
Melvin, we are still waiting for proposals

**Comment by pauldriggers911** [comment]
Hello
Currently, seems that the product version for Plaid is being run on local dev env. How can run the staging version on local dev env?

**Comment by pauldriggers911** [comment]
Contributor details
Your Expensify account email: pauldriggers911@outlook.com
Upwork Profile Link: https://www.upwork.com/freelancers/~010d3ea472a2944d4e

**Comment by s77rt** [comment]
@pauldriggers911 Go to Settings -> Preferences and switch on the "Use Staging Server" switch. For follow up questions please ask on slack `#expensify-open-source` channel. If you don't have slack access please checkout the [contributing guide](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#slack-channels).

**Comment by pauldriggers911** [comment]
Hello, @isagoico. Will you provide full video for this issue? I am not sure which step you were facing it in.

**Comment by Christinadobrzyn** [comment]
Reporting another issue that I think is impacting the testing fo this one - https://expensify.slack.com/archives/C049HHMV9SM/p1687475426113199

I'll follow this ^ chat thread and provide more information

**Comment by Christinadobrzyn** [comment]
https://github.com/Expensify/App/issues/21461 - I think this might be blocking testing of this issue. @s77rt @pauldriggers911 let me know if you agree!

**Comment by s77rt** [comment]
@Christinadobrzyn Yes, it seems to be the case. Should we put this on hold?


https://github.com/Expensify/App/assets/16493223/8f8b5126-9b02-41b3-9979-335971226d5b

**Comment by Christinadobrzyn** [comment]
Yes - thanks for testing. Let's put this on hold for https://github.com/Expensify/App/issues/21461

**Comment by Christinadobrzyn** [comment]
on hold for https://github.com/Expensify/App/issues/21461

**Comment by Christinadobrzyn** [comment]
on hold for https://github.com/Expensify/App/issues/21461

**Comment by Christinadobrzyn** [comment]
I'm going to be ooo until August 31st so going to unassign and assign a new teammate. 

At this time, we're on hold for https://github.com/Expensify/App/issues/21461

**Comment by maddylewis** [comment]
moving to weekly since it's on hold

**Comment by maddylewis** [comment]
i believe we are still holding - https://github.com/Expensify/App/issues/21461

**Comment by Christinadobrzyn** [comment]
Back from ooo - thanks for handling this @maddylewis I can take it off your plate

**Comment by s77rt** [comment]
Still on hold

**Comment by Christinadobrzyn** [comment]
Hey @s77rt this is now closed - https://github.com/Expensify/App/issues/21461

I just tested our OP steps again and I'm seeing this error so I wonder if this is expected or if something else needs to happen before working on this? 

<img width="1200" alt="image" src="https://github.com/Expensify/App/assets/51066321/a656ce12-9865-4dc4-a6aa-a2369cd84ad2">

**Comment by s77rt** [comment]
@Christinadobrzyn Since https://github.com/Expensify/App/issues/21461 was closed as not to be fixed, I think we can close this one too as it's also related to the "Pay with Expensify".

cc @marcaaron

**Comment by Christinadobrzyn** [comment]
Sounds good! let's close this one for now and we can reopen it if needed.
